src/store/transactions/exit/exit.thunks.ts

```diff
@@ -146,13 +145,12 @@ function fetchEstimatedWithdrawFee(token: Token, amount: BigNumber) {
         : BigNumber.from(gasLimit);
 
       if (tokensPriceTask.status === "successful" || tokensPriceTask.status === "reloading") {
-        const tokenUSD = tokensPriceTask.data[ETHER_TOKEN_ID].USD;
-        const feeUSD = Number(formatEther(feeBigNumber)) * tokenUSD;
+        const ethToken = tokensPriceTask.data[ETHER_TOKEN_ID];
```

Comment:
Good catch! This is, in fact, wrong. Because `tokensPriceTask.data` is an array, and we should find by `token.id`. I've updated both cases (exit and withdraw thunks).